openedx_external_enrollments/external_enrollments.py

```diff
@@ -0,0 +1,424 @@
+"""External enrollments file."""
```

Comment:
Isn't better if each the enrollments types is in its own file? @andrey-canon

Reply:
do you think it is better a folder called external enrollments and a individual file for each class ?

Reply:
Definitely. @andrey-canon